src/main/utils/shell-env/env.injectable.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import process from "process";
+
+const processEnvInjectable = getInjectable({
+  id: "process-env",
+  instantiate: () => process.env,
+  causesSideEffects: true,
```

Comment:
Why is this true? Because process.env can be modified? So every value that can be mutated should have `causesSideEffects` true?

Reply:
`causesSideEffects` is a hint for unit testing only. It means that if this injectable is not overridden in a unit test then it could cause subtle test failures by accidentally sharing data between tests.